<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreManagerWindow</class>
 <widget class="QMainWindow" name="StoreManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="storeManagerTabWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>1031</width>
      <height>621</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="myStoreTab">
     <attribute name="title">
      <string>我的商铺</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>461</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>😘我的商铺</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addStoreButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>510</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>添加商铺</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteStoreButton">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>510</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>删除商铺</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="myStoreTableWidget">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>91</y>
        <width>1031</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="orderMessageTab">
     <attribute name="title">
      <string>订单信息</string>
     </attribute>
     <widget class="QTableWidget" name="orderInformationTableWidget">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>101</y>
        <width>1031</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>401</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>速速接单！🥵🥵🥵</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
